src/sentry/workflow_engine/migration_helpers/alert_rule.py

```diff
@@ -529,3 +583,45 @@ def dual_delete_migrated_alert_rule(
         RegionScheduledDeletion.schedule(instance=data_source, days=0, actor=user)
 
     return
+
+
+def dual_delete_migrated_alert_rule_trigger(
+    alert_rule_trigger: AlertRuleTrigger,
+    user: RpcUser | None = None,
+) -> None:
+    priority = (
```

Comment:
I notice we have the same logic in `migrate_metric_action` for `priority`, could be pulled out into a const